CONTRIBUTING.md

```diff
@@ -2,192 +2,176 @@
 
-If you rename an article, insert an anchor so that the old link still works:
+If you rename a section, the link to that section might break. To avoid this, add an anchor manually:
 
 ```html
-<div id="some-title"></div>
+<div id="old-section-title"></div>
 ```
 
-### Tags
+This will allow existing links to continue working even after the section title is changed.
 
-To distinguish original and translatory articles, they should be marked with corresponding tags. For original articles, it's
+### Article Tags
 
-```md
----
-tags:
-    - Original
----
-```
-
-And for translated articles, it's
+We use tags to differentiate between original content and translated articles. Add the appropriate tag at the top of your article:
 
-```md
----
-tags:
-    - Translated
-e_maxx_link: ...
----
-```
+- **For original articles**:
 
-Here, instead of `...` one should place the last part of the link to the original article. E.g. for [Euler function article](http://e-maxx.ru/algo/euler_function) it should be
+    ```md
+    ---
+    tags:
+        - Original
+    ---
+    ```
 
+- **For translated articles**:
 
-```md
----
-tags:
-    - Translated
-e_maxx_link: euler_function
----
-```
+    ```md
+    ---
+    tags:
+        - Translated
+    e_maxx_link: <original-link>
+    ---
+    ```
 
+    Replace `<original-link>` with the last part of the URL (e.g., for `http://e-maxx.ru/algo/euler_function`, use `euler_function`).
 
-## Some conventions
+## Conventions
 
-* We have agreed as of issue [#83](https://github.com/cp-algorithms/cp-algorithms/issues/83) to express binomial coefficients with `\binom{n}{k}` instead of `C_n^k`. The first one renders as $\binom{n}{k}$ and is a more universal convention. The second would render as $C_n^k$.
+We follow certain conventions across the project. For example, we agreed to use the `\binom{n}{k}` notation for binomial coefficients instead of `C_n^k` as outlined in [issue #83](https://github.com/cp-algorithms/cp-algorithms/issues/83).
 
 ## Adding Problems
 
-Try to add problems in ascending order of their difficulty. If you don't have enough time to do so, still add the problem. Lets hope that the next person will sort them accordingly.
+When adding problems, try to arrange them by difficulty. If you're unable to, don't worry—just add the problem, and someone else can adjust the order later.
 
-## Local development
+## Local Development Setup
 
-You can render the pages locally. All you need is Python, with the installed `mkdocs-material` package.
+You can preview changes locally before pushing them to GitHub. To do this:
 
-```console
-$ git clone --recursive https://github.com/cp-algorithms/cp-algorithms.git && cd cp-algorithms
-$ scripts/install-mkdocs.sh # requires pip installation
-$ mkdocs serve
-```
+1. Clone the repository:
 
-Note that some features are disabled by default for local builds.
+    ```console
+    git clone --recursive https://github.com/cp-algorithms/cp-algorithms.git && cd cp-algorithms
```

Comment:
Please also remove prefix `$` in all other places in the article where applicable.